idgenerator/idgenerator.go

```diff
@@ -0,0 +1,116 @@
+/*
+Package idgenerator contains several Span and Trace ID generators which can be
+used by the Zipkin tracer. Additional third party generators can be plugged in
+if they adhere to the IDGenerator interface.
+*/
+package idgenerator
+
+import (
+	"math/rand"
+	"sync"
+	"time"
+
+	"github.com/openzipkin/zipkin-go/model"
+)
+
+var (
+	seededIDGen = rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
Not sure whether this is a global per each service run, for me it makes sense to use this as a method rather than a variable.

Reply:
this is a very important piece of code to make sure each application instance gets a different seed... since we allow multiple tracers per service and don't want multiple random generator sources for tracers, it doesn't belong in tracer creation. At the same time I don't want to rely on an implementor to not forget seeding the generators.

Reply:
I do agree, I was more thinking as this as a function instead of a variable but as I said, not a strong opinion about it.